prod/westeurope/eucovidcert/functions_eucovidcert/function_app/terragrunt.hcl

```diff
@@ -104,6 +124,7 @@ inputs = {
     dependency.subnet_apimapi.outputs.id,
     dependency.subnet_appbackendl1.outputs.id,
     dependency.subnet_appbackendl2.outputs.id,
+    dependency.subnet_fnservices.outputs.id,
```

Comment:
```suggestion
```
@balanza fn services is the client so fnservice needs to whitelist fneucertcovid' subnet. This rule is already setted in https://github.com/pagopa/io-infrastructure-live-new/pull/535/commits/efdc6f9ae33091b39375e2d9791a68cac3dba98b

Reply:
you're right, my bad